data/sources/admin-boundaries.json

```diff
@@ -45,12 +45,17 @@
     {
       "id": "cdtas",
       "sql": "SELECT the_geom_webmercator, cdtaname FROM nycdta2020",
-      "dataPipeline": false
+      "dataPipeline": true
+    },
+    {
+      "id": "factfinder_dcp_community_districts",
+      "sql": "SELECT the_geom_webmercator, borocd, borocd AS geolabel, borocd as geoid FROM nycd2020",
+      "dataPipeline": true
```

Comment:
These should probably be `false` since they're not maintained by GIS/EDM?

Reply:
I don't know what that variable is supposed to represent.  Should I change all of the ones used by factfinder to "dataPipeline": false?

Reply:
If the SQL in the `sql` property is querying a shapefile you or Labs/EDM just recently uploaded, it should be `false`. If it is a dataset that is maintained by GIS+EDM, it should be true.